### PR TITLE
Update the user guide

### PR DESCRIPTION
**Pull request by Wongqingyee**
Updated help, add, archive, edit_archived, delete_archived, unarchived, clear_archived

### CODE REVIEW

**Review by ronnnnnnnnn — COMMENTED** [review]
I think the Table of Contents and the header numbers after 7.9 need to be updated.

**Comment by Wongqingyee** [comment]
> I think the Table of Contents and the header numbers after 7.9 need to be updated.

Yup. I will leave that to yashma to change it, to prevent any changes in authorship, since she is doing the toc.

**Comment by willowisp01** [comment]
> > I think the Table of Contents and the header numbers after 7.9 need to be updated.
> 
> Yup. I will leave that to yashma to change it, to prevent any changes in authorship, since she is doing the toc.

I think it's fine if you DIY because she will have to check what we wrote individually. Which is basically just extra work since we already know where to put it.

File: docs/images/UnarchivedUI.png
Comment (yashma-sonara) [inline docs/images/UnarchivedUI.png:1]: The images are quite clear :)

**Review by yashma-sonara — APPROVED** [review]
LGTM

**Pull request merged**